libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1079,15 +1089,8 @@ bool AP_AHRS_DCM::airspeed_estimate(float &airspeed_ret) const
 //  - otherwise from a cached wind-triangle estimate value (but returning false)
 bool AP_AHRS_DCM::airspeed_estimate(uint8_t airspeed_index, float &airspeed_ret) const
 {
-    if (airspeed_sensor_enabled(airspeed_index)) {
-        airspeed_ret = AP::airspeed()->get_airspeed(airspeed_index);
-    } else if (AP::ahrs().get_wind_estimation_enabled() && have_gps()) {
-        // estimate it via GPS speed and wind
-        airspeed_ret = _last_airspeed;
-    } else {
-        // give the last estimate, but return false. This is used by
-        // dead-reckoning code
-        airspeed_ret = _last_airspeed;
+    // Return false if we are using the previous airspeed estimate
```

Comment:
Add a comment here re-inforcing the fact that `airspeed_ret` is always filled in and why.

Reply:
Done though it is verbose ;) If you want it shorter umm I'll gladly take what you think should be there